pkg/apis/pipeline/v1beta1/resource_types.go

```diff
@@ -23,12 +23,12 @@ import (
 	"github.com/hashicorp/go-multierror"
 )
 
-// PipelineResourceResult is used to write key/value pairs to TaskRun pod termination messages.
+// TaskAndTerminationResult is used to write key/value pairs to TaskRun pod termination messages.
```

Comment:
I'm struggling to think of a good name here: it's the internal representation of TaskRunResults and TektonInternalResults (and is not necessarily written to termination messages, in the case of sidecar logs). Maybe `GenericResult`? idk

Reply:
Thanks, renamed to RunResult and aliased for backward compatibility. 
I was not sure if this Struct should also be moved to result_types.go or somewhere else? I am thinking of a struct in an unversioned package and aliased in v1beta1 if that makes sense.

Reply:
I think that's fine to leave for a follow-up PR